tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -52,14 +54,18 @@ def _test_approach(env_name, approach_name, excluded_predicates="",
     approach2.load()
     if try_solving:
         approach2.solve(task, timeout=CFG.timeout)
+    utils.update_config({"env": env_name, "approach": approach_name,
+                         "seed": 12345})
 
 
 def test_nsrt_learning_approach():
     """Tests for NSRTLearningApproach class.
     """
     _test_approach(env_name="cover", approach_name="nsrt_learning")
+    # Sampler learning requires more data than we are allowing for fast
+    # unit tests, so don't try solving.
     _test_approach(env_name="cover_multistep_options",
-                   approach_name="nsrt_learning")
+                   approach_name="nsrt_learning", try_solving=False)
```

Comment:
After you rebase to most recent master, I'd recommend also adding `sampler_learner="random"` in addition to `try_solving=False` for extra speed.